### PR TITLE
[tk_multi_publish2] Merged latest master changes from fork.

### PR DESCRIPTION
**Pull request by cblanchette97**

### CODE REVIEW

File: info.yml [hunk 1]
```diff
@@ -102,24 +118,31 @@ configuration:
...
-# More verbose description of this item 
+# More verbose description of this item
 display_name: "Publish"
 description: "Provides UI and functionality to publish files to Shotgun."
 
 # Required minimum versions for this item to run
 requires_shotgun_version:
-requires_core_version: "v0.18.153"
-requires_engine_version: 
+requires_core_version: "v0.19.4"
```
Comment (SComteSqueeze) [inline info.yml:138]: Whew, that was a close call :)

Comment (cblanchette97) [reply info.yml:138]: Oh my yes, I didn't see that ahaha

**Comment by SComteSqueeze** [comment]
That looks good!

**Comment by cblanchette97** [comment]
Request will be merged monday morning, after the official pipeline release!

**Pull request merged**